bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/AccessibleToolItemBuilder.java

```diff
@@ -0,0 +1,61 @@
+package org.eclipse.ui.texteditor;
```

Comment:
Copyright header is missing

Reply:
addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/0c0d22bc8f0ea13eb6e1124c89747f7a4f02e266